sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -1258,3 +1242,71 @@ def test_gaussian_mixture_setting_best_params():
         "lower_bound_",
     ]:
         assert hasattr(gmm, attr)
+
+
+@pytest.mark.parametrize(
+    "init_params", ["random", "random_from_data", "k-means++", "kmeans"]
+)
+def test_init_param(init_params):
+    # Check that the init_param options produces valid output.
+    # Compare all to default (kmeans).
```

Comment:
I think this test is brittle since afaik there's no guarantee that different inits will converge to the same solution (even up to a permutation). I'd be in favor of not adding such a test

Reply:
Should we delete this test outright or leave it for another PR? FWIW, I think you are right but I'd have to read a bit/ask around to check whether that assumption on GMM convergence under different initialization methods is valid.

Reply:
I checked with a wide range of random seeds at it fails from time to time which confirms that it's not a property we can assume.  So I definitely think that we should not include it in this PR

Reply:
Thanks! Just removed it.